### DISCUSSION TITLE
what if I want to increase or decrease worker / master nodes when deployment already done?

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [vitobotta/hetzner-k3s#333] [Q&A] [answered]
what if I want to increase or decrease worker / master nodes ?
do I just edit the config if I want to increase and re deploy and will it mess up my runnign pods in the process? 
or how do we increase?  or it's autoincreased on load and what if hetzner accuont have limits on server like 5 server limit and I have to increase that also. 
thanks let's discuse

**Comment by vitobotta** [comment, marked as answer]
It's easy! All you have to do is just increase the istance count and re-run the create command. It will automatically detect what servers you already have and create the missing ones. 

Removing nodes is a bit more involved, see https://github.com/vitobotta/hetzner-k3s?tab=readme-ov-file#scaling-down-a-node-pool.

I usually like to have one or two static node pools for some things, and have one or more nodepools with the autoscaling enabled, so I don't need to worry about adding and removing nodes myself. Have you tried that?

**Reply by 0xMALVEE** [reply]
thanks
